src/main/resources/Ideas/RecentIdeasMacro.xml

```diff
@@ -0,0 +1,269 @@
+      </contentType>
+      <defaultCategory>
+        <disabled>0</disabled>
+        <name>defaultCategory</name>
+        <number>4</number>
+        <prettyName>Default category</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </defaultCategory>
+      <description>
+        <disabled>0</disabled>
+        <name>description</name>
+        <number>3</number>
+        <prettyName>Macro description</prettyName>
+        <rows>5</rows>
+        <size>40</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </description>
+      <id>
+        <disabled>0</disabled>
+        <name>id</name>
+        <number>1</number>
+        <prettyName>Macro id</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </id>
+      <name>
+        <disabled>0</disabled>
+        <name>name</name>
+        <number>2</number>
+        <prettyName>Macro name</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </name>
+      <supportsInlineMode>
+        <disabled>0</disabled>
+        <displayFormType>select</displayFormType>
+        <displayType>yesno</displayType>
+        <name>supportsInlineMode</name>
+        <number>5</number>
+        <prettyName>Supports inline mode</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </supportsInlineMode>
+      <visibility>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <multiSelect>0</multiSelect>
+        <name>visibility</name>
+        <number>6</number>
+        <prettyName>Macro visibility</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator>|</separator>
+        <separators>|</separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>Current User|Current Wiki|Global</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </visibility>
+    </class>
+    <property>
+      <code>{{velocity}}
+#set($discard = $doc.use('Ideas.IdeasClass'))
+#set($discard = $xwiki.jsx.use('Ideas.IdeasClass'))
+#set($discard = $xwiki.ssx.use('Ideas.IdeasSheet'))
+#set($discard = $services.localization.use('document', 'Ideas.IdeasTranslations'))
+##
+#set($hql = ", BaseObject obj where doc.fullName=obj.name and obj.className='Ideas.IdeasClass' and doc.name!='IdeasTemplate' order by doc.creationDate desc")
+#set($results = $services.query.hql($hql).execute())
+#set($i = 0)
+#set ($obj = $document.getObject('Ideas.IdeasClass'))
+##
+{{html wiki="true"}}
```

Comment:
Yes. Otherwise the buttons for voting aren't displayed properly.
![screen shot 2015-01-23 at 15 30 44](https://cloud.githubusercontent.com/assets/8993426/5875364/ea330788-a314-11e4-9b68-d379c0afabc6.png)